Makefile

```diff
@@ -236,7 +236,7 @@ shell:
 	bash
 
 configs-integration-test:
-	/bin/bash -c "go test -v -tags 'netgo integration' -timeout 30s ./pkg/configs/... ./pkg/ruler/..."
+	/bin/bash -c "go test -v -tags 'netgo integration' -timeout 60s ./pkg/configs/... ./pkg/ruler/..."
```

Comment:
Never a good sign... but ok

Reply:
Like I mentioned in the earlier comment, the tests did not check for active alerts and so the tests can immediately call GetRules. Rules need to be evaluated for alerts to be present and it takes extra time to evaluate the rules. `configs-integration-test` are also calling the ruler tests which takes extra time to evaluate the rules. Basically it is a cascading effect. Even if we remove the sleep, which I did in the recent version, some of the tests are going to take longer time for alerts to be populated

Reply:
I will look at this again later in the week and see if there is a way to better organize the test or create a new test but even in this case, the tests would need to run longer for alerts to be populated